src/periodic_table.py

```diff
@@ -0,0 +1,359 @@
+    '74': {'atomic_num': 74, 'symbol': 'W', 'name': 'Tungsten', 'group': 6, 'period': 6, 'atomic_weight': 183.84,
+           'density': 19.25, 'melting_point': 3695, 'boiling_point': 5828, 'heat_capacity': 0.132,
+           'electronegativity': 2.36, 'precise': True, 'type': METAL},
+    '75': {'atomic_num': 75, 'symbol': 'Re', 'name': 'Rhenium', 'group': 7, 'period': 6, 'atomic_weight': 186.207,
+           'density': 21.02, 'melting_point': 3459, 'boiling_point': 5869, 'heat_capacity': 0.137,
+           'electronegativity': 1.9, 'precise': True, 'type': METAL},
+    '76': {'atomic_num': 76, 'symbol': 'Os', 'name': 'Osmium', 'group': 8, 'period': 6, 'atomic_weight': 190.23,
+           'density': 22.61, 'melting_point': 3306, 'boiling_point': 5285, 'heat_capacity': 0.13,
+           'electronegativity': 2.2, 'precise': True, 'type': METAL},
+    '77': {'atomic_num': 77, 'symbol': 'Ir', 'name': 'Iridium', 'group': 9, 'period': 6, 'atomic_weight': 192.217,
+           'density': 22.56, 'melting_point': 2719, 'boiling_point': 4701, 'heat_capacity': 0.131,
+           'electronegativity': 2.2, 'precise': True, 'type': METAL},
+    '78': {'atomic_num': 78, 'symbol': 'Pt', 'name': 'Platinum', 'group': 10, 'period': 6, 'atomic_weight': '195.084*9',
+           'density': 21.46, 'melting_point': 2041.4, 'boiling_point': 4098, 'heat_capacity': 0.133,
+           'electronegativity': 2.28, 'precise': True, 'type': METAL},
+    '79': {'atomic_num': 79, 'symbol': 'Au', 'name': 'Gold', 'group': 11, 'period': 6, 'atomic_weight': 196.96657,
+           'density': 19.282, 'melting_point': 1337.33, 'boiling_point': 3129, 'heat_capacity': 0.129,
+           'electronegativity': 2.54, 'precise': True, 'type': METAL},
+    '80': {'atomic_num': 80, 'symbol': 'Hg', 'name': 'Mercury', 'group': 12, 'period': 6, 'atomic_weight': 200.592,
+           'density': 13.5336, 'melting_point': 234.43, 'boiling_point': 629.88, 'heat_capacity': 0.14,
+           'electronegativity': 2, 'precise': True, 'type': METAL},
+    '81': {'atomic_num': 81, 'symbol': 'Tl', 'name': 'Thallium', 'group': 13, 'period': 6, 'atomic_weight': 204.38,
+           'density': 11.85, 'melting_point': 577, 'boiling_point': 1746, 'heat_capacity': 0.129,
+           'electronegativity': 1.62, 'precise': True, 'type': METAL},
+    '82': {'atomic_num': 82, 'symbol': 'Pb', 'name': 'Lead', 'group': 14, 'period': 6, 'atomic_weight': 207.2,
+           'density': 11.342, 'melting_point': 600.61, 'boiling_point': 2022, 'heat_capacity': 0.129,
+           'electronegativity': 1.87, 'precise': True, 'type': METAL},
+    '83': {'atomic_num': 83, 'symbol': 'Bi', 'name': 'Bismuth', 'group': 15, 'period': 6, 'atomic_weight': 208.9804,
+           'density': 9.807, 'melting_point': 544.7, 'boiling_point': 1837, 'heat_capacity': 0.122,
+           'electronegativity': 2.02, 'precise': True, 'type': METAL},
+    '84': {'atomic_num': 84, 'symbol': 'Po', 'name': 'Polonium', 'group': 16, 'period': 6, 'precise': False,
```

Comment:
Precise is the column we talked about when working with elements still under research which have some approximate values right?